README.md

```diff
@@ -105,6 +106,18 @@ Depending on the example you need to export needed environment variables for API
    php examples/toolchain-serpapi.php
    ```
 
+1. Weather Tool
+   ```bash
+   export OPENAI_API_KEY=sk-...
+   php examples/toolchain-weather.php
```

Comment:
shouldn't we talk consistently about `toolbox` instead of `toolchain`?

Reply:
Yea, good point, already legacy after that universal chain refactoring